iocore/net/UnixNetVConnection.cc

```diff
@@ -1296,7 +1294,7 @@ UnixNetVConnection::connectUp(EThread *t, int fd)
   }
 
   if (fd == NO_FD) {
-    res = con.connect(&server_addr.sa, options);
+    res = con.connect(NULL, options);
```

Comment:
In UnixNetProcessor::connect_re_internal(), the con.addr.sa set with target addr.
The member server_addr is removed from NetVConnection in this PR.